include/z64eff_footmark.h

```diff
@@ -0,0 +1,33 @@
+#ifndef Z64EFF_FOOTMARK_H
+#define Z64EFF_FOOTMARK_H
+
+#include "ultra64.h"
+#include "z64math.h"
+
+struct PlayState;
+struct Actor;
+
+typedef struct EffFootmark {
+    /* 0x00 */ MtxF displayMatrix;
+    /* 0x40 */ struct Actor* actor;
+    /* 0x44 */ Vec3f location;
```

Comment:
```suggestion
    /* 0x44 */ Vec3f pos;
```
This is just position right? Same for the arg in `EffFootmark_Add`?